plugins/by-name/roslyn/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, pkgs, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "roslyn";
+  packPathName = "roslyn.nvim";
+  package = "roslyn-nvim";
+
+  maintainers = [ lib.maintainers.GaetanLepage ];
+
+  extraConfig = cfg: {
+    extraPlugins = [ pkgs.vimPlugins.roslyn-nvim ];
```

Comment:
```suggestion
```

`mkNeovimPlugin` already does this; that's what the `package` argument is for above.

Reply:
Will do when I get internet back on my desktop machine.

Thanks!